lms/djangoapps/courseware/management/commands/dump_to_neo4j.py

```diff
@@ -0,0 +1,184 @@
+"""
+This file contains a management command for exporting the modulestore to
+neo4j, a graph database.
+"""
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from py2neo import Graph, Node, Relationship
+from py2neo.compat import integer, string, unicode as neo4j_unicode
+from request_cache.middleware import RequestCache
+from xmodule.modulestore.django import modulestore
+
+log = logging.getLogger(__name__)
```

Comment:
we use [`log`](https://github.com/edx/edx-platform/search?utf8=%E2%9C%93&q=%22log+%3D+logging.getLogger%22&type=Code) more often then we do [`logger`](https://github.com/edx/edx-platform/search?utf8=%E2%9C%93&q=%22logger+%3D+logging.getLogger%22&type=Code), so I think it makes sense to stick with log.